en/developer_guide/elasticsearch_add_new_indexed_document.rst

```diff
@@ -0,0 +1,85 @@
+Elasticsearch add new indexed document
+======================================
+
+General note
+------------
+
+To use this documentation, you should already have installed `Elasticsearch`_ on your
+environment.
+
+This documentation will describe how to create the schema, and how to populate the index of
+the new document.
+
+Schema creation
+---------------
+
+To generate the schema of your document, you need to create an initializer that will implement
+``ElasticaSchemaInitializer``.
+
+This initializer should be tagged with :
+
+.. code-block:: yaml
+
+    tags:
+        - { name: open_orchestra_elastica.schema_initializer.strategy }
+
+This initializer will be called by the console command ``orchestra:elastica:schema:create``.
```

Comment:
What happens if several services are tagged with open_orchestra_elastica.schema_initializer.strategy ? What is the criteria to choose one or another strategy ?